usagov/vote-gov-drupal
# VOTE-977: Remove vote_migrate module.

## Jira ticket

[VOTE-977](https://cm-jira.usa.gov/browse/VOTE-977)

## Description

Remove unused modules.

## Deployment and testing

### Post-deploy steps

n/a

### QA/Testing instructions

1. Run `lando retune`
2. Confirm that config import runs without any issues with dependency configurations not finding an enabled module.

## Checklist for the Developer




## Checklist for the Peer Reviewers
